### PR TITLE
Introduces EC2 Install

### PR DESCRIPTION
**Pull request by edwardcallahan**

### CODE REVIEW

**Comment by huntc** [comment]
Need to update the .toc file otherwise it won't be able to be referenced.

File: src/main/play-doc/intro/Install-EC2.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+* Amazon Web Services account with EC2 admin rights
+* Debian package of ConductR
+
+### Preparing EC2
+
+Begin by preparing the EC2 network and security environment. This tutorial uses a Virtual Private Cloud (VPC) with a Classless Inter-Domain Routing (CIDR) of 10.0.0.0/16 and example addresses will be based accordingly.
+
+#### Subnets and Security Groups
+For better resilience, deploy nodes across multiple availability zones (AZ). Create three subnets in the VPC. Place each subnet in a different availability zone by specifying an AZ during subnet creation. Our example subnets names indicate their AZ. They aare SN-A with a CIDR of 10.0.1.0/24, SN-B with a CIDR of 10.0.2.0/24 and SN-C with a CIDR of 10.0.3.0/24. Each subnet will need an internet gateway added to their route table for the destination 0.0.0.0/0. This required so that our nodes can access the internet. All subnets can use the same route table.
```
Comment (huntc) [inline src/main/play-doc/intro/Install-EC2.md]: aare

File: src/main/play-doc/intro/Install-EC2.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+#### Installing JRE 8
+
+Install Java 8 as the default JRE. You will need to accept the Oracle license agreement.
+
+```bash
+sudo add-apt-repository -y ppa:webupd8team/java 
+sudo apt-get update 
+sudo apt-get -y install oracle-java8-installer && sudo apt-get clean
+sudo apt-get -y install oracle-java8-set-default
+echo "JAVA_HOME=/usr/lib/jvm/java-8-oracle" | sudo tee -a /etc/environment
```
Comment (huntc) [inline src/main/play-doc/intro/Install-EC2.md:63]: Could I get you to update the general installation page with instructions to install Java 8, or make this consistent with the general one. The general one presently reads:

> First ensure that you have the default-jre package installed (the default is JRE 7 at the time of writing):
> 
> sudo apt-get install default-jre

...and so forth.

I was careful about that install procedure as I don't want to violate the official Oracle supported means of doing things. However given that your above approach requires the user to accept a license agreement then I think we're good.

We just need to be consistent. :-)

File: src/main/play-doc/intro/Install-EC2.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+```
+#### Installing ConductR 
+
+The tutorial assumes that you have obtained the `conductr_%PLAY_VERSION%_all.deb` Debian package. 
+
+Scp the ConductR installation package to the image host and install ConductR as any other Debian package.
```
Comment (huntc) [inline src/main/play-doc/intro/Install-EC2.md]: "Secure copy (`scp`) the ConductR..."

**Comment by huntc** [comment]
Great! Small style comment: when referencing ports or ip addresses the convention we've used is to `10.0.2.20` them i.e. monospace them withing backquotes.

**Comment by edwardcallahan** [comment]
Thanks for the review. Updated accordingly.

**Comment by huntc** [comment]
Thanks!

**Comment by huntc** [comment]
...and quick as a flash: http://conductr.typesafe.com/

**Pull request merged**